rpc.go

```diff
@@ -247,6 +197,35 @@ func (r *rpc) Clear(in *kvv1.Request, _ *kvv1.Response) error {
 	return nil
 }
 
+func getStorage(in *kvv1.Request, span trace.Span, r *rpc, op errors.Op) (kv.Storage, error) {
```

Comment:
No need to pass span, rpc and op.
Span should be recorded on exit.
*rpc -> method getStorage should be the *rpc structure
op - should not be passed from outside, because it is a local operation.